Script/PRONTO.py

```diff
@@ -1455,42 +1511,29 @@ def main(argv):
 				slide6_table_height = 1.63
 				slide6_table_font_size = 7
 				if_print_rowNo = False
+				table6_column_width = []
 				for table_index in slide6_table_ppSlide:
-					slide6_table_nrows = insert_table_to_ppt(slide6_table_data_file,table_index,slide6_table_name,slide6_header_left,slide6_header_top,slide6_header_width,slide6_table_left,slide6_table_top,slide6_table_width,slide6_table_height,slide6_table_font_size,slide6_table_header,output_ppt_file,if_print_rowNo)
+					slide6_table_nrows = insert_table_to_ppt(slide6_table_data_file,table_index,slide6_table_name,slide6_header_left,slide6_header_top,slide6_header_width,slide6_table_left,slide6_table_top,slide6_table_width,slide6_table_height,slide6_table_font_size,slide6_table_header,slide6_table_header_aliase,output_ppt_file,if_print_rowNo,table6_column_width)
 				output_file_preMTB_AppendixTable = output_file_preMTB_table_path + "_preMTBTable_Appendix.txt"
 				output_table_file_filterResults_AllReporVariants_CodingRegion = output_file_preMTB_table_path + "_AllReporVariants_CodingRegion.txt"
 				stable_text = update_ppt_variant_summary_table(slide6_table_nrows,DNA_sampleID,RNA_sampleID,TMB_DRUP,TMB_DRUP_str,DNA_variant_summary_file,RNA_variant_summary_file,output_file_preMTB_AppendixTable,output_table_file_filterResults_AllReporVariants_CodingRegion,output_ppt_file)
-				output_file_preMTB_VigdisTable = output_file_preMTB_table_path + "_preMTBTable_Vigdis.txt"
+				output_file_sequence_summary_table = output_file_preMTB_table_path + "_sequence_summary.txt"
                 
-				# Slide8 Table1: Sequence data summary: protein coding altering cariants*
-				slide8_table_data_file = output_file_preMTB_VigdisTable
+				# Slide8 Table: Sequence data summary: Variants alter protein code*
+				slide8_table_data_file = output_file_sequence_summary_table
 				slide8_table_ppSlide = 9
-				slide8_table_name = "Table1: Sequence data summary: protein coding altering variants*  " + "  TSO500 " + pipline + " "
-				slide8_header_left = 0.19
-				slide8_header_top = 0.31
-				slide8_header_width = 6.98
+				slide8_table_name = "Sequence data summary for " + DNA_sampleID + ": Variants that alter protein code and splice sites." + "  TSOPPI Version string: " + pipline + " "
+				slide8_header_left = 0.25
```

Comment:
can these numbers be calculated somehow? It seems like this will always change a bit according to certain parameters and it would make more sense to just have PRONTO adjust them on the go.

Reply:
Let's skip this for now